src/test/app/PayChan_test.cpp

```diff
@@ -870,9 +870,221 @@ struct PayChan_test : public beast::unit_test::suite
     }
 
     void
-    testRPC ()
+    testAccountChannelsRPC()
     {
-        testcase ("RPC");
+        testcase("AccountChannels RPC");
+
+        using namespace jtx;
+        using namespace std::literals::chrono_literals;
+        Env env(*this);
+        auto const alice = Account("alice");
+        auto const bob = Account("bob");
+        auto const charlie = Account("charlie", KeyType::ed25519);
+        env.fund(XRP(10000), alice, bob, charlie);
+        auto const pk = alice.pk();
+        auto const settleDelay = 3600s;
+        auto const channelFunds = XRP(1000);
+        env(create(alice, bob, channelFunds, settleDelay, pk));
+        env.close();
+        auto const chan1Str = to_string(channel(*env.current(), alice, bob));
+        std::string chan1PkStr;
+        {
+            auto const r =
+                env.rpc("account_channels", alice.human(), bob.human());
+            BEAST_EXPECT(r[jss::result][jss::channels].size() == 1);
+            BEAST_EXPECT(
+                r[jss::result][jss::channels][0u][jss::channel_id] == chan1Str);
+            BEAST_EXPECT(r[jss::result][jss::validated]);
+            chan1PkStr =
+                r[jss::result][jss::channels][0u][jss::public_key].asString();
+        }
+        {
+            auto const r = env.rpc("account_channels", alice.human());
+            BEAST_EXPECT(r[jss::result][jss::channels].size() == 1);
+            BEAST_EXPECT(
+                r[jss::result][jss::channels][0u][jss::channel_id] == chan1Str);
+            BEAST_EXPECT(r[jss::result][jss::validated]);
+            chan1PkStr =
```

Comment:
you assign this value (twice) bet never do anything with it...

Reply:
Fixed in `ee122e24b [fold] rm write-only var`